tests/cross_functional/scale/test_multiple_storage_coexistence.py

```
@@ -0,0 +1,326 @@
+            ocp_obj = OCP(kind=f"{kind}.scale.spectrum.ibm.com", namespace=ns)
+            try:
+                items = ocp_obj.get().get("items", [])
+                for res in items:
+                    name = res["metadata"]["name"]
+                    log.info(f"Scrubbing {kind}: {name}")
+                    exec_cmd(
+                        f"oc patch {kind.lower()}.scale.spectrum.ibm.com {name} -n {ns}"
+                        f' --type=merge -p \'{{"metadata":{{"finalizers":null}}}}\''
+                    )
+                    ocp_obj.delete(resource_name=name)
+            except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)